source/apps.html.haml.markdown

```diff
@@ -30,6 +30,75 @@ description: Applications distributed as Flatpaks, ready to download.
   <span class="unselectable">$ </span>flatpak run org.telegram.TelegramDesktopDevel
   </pre>
 
+  ## <a name="spotify"></a>Spotify
+
+  Builds of the [Spotify](https://spotify.com/) desktop client are being made available. To install, run:
+
+  <pre>
+  <span class="unselectable">$ </span>flatpak install --from https://s3.amazonaws.com/alexlarsson/spotify-repo/spotify.flatpakref
+  <span class="unselectable">$ </span>flatpak run com.spotify.Client
+  </pre>
+
+  ## <a name="skype"></a>Skype
+
+  Builds of the alpha version of the [Skype](https://skype.com/) desktop client are being made available. To install, run:
+
+  <pre>
+  <span class="unselectable">$ </span>flatpak install --from https://s3.amazonaws.com/alexlarsson/skype-repo/skype.flatpakref
+  <span class="unselectable">$ </span>flatpak run com.skype.Client
+  </pre>
+
+  ## <a name="monodevelop"></a>MonoDevelop
+
+  Preview release of MonoDevelop as a Flatpak is available here:
+
+  <pre>
+  <span class="unselectable">$ </span>flatpak install --user --from https://download.mono-project.com/repo/monodevelop.flatpakref
+  <span class="unselectable">$ </span>flatpak run com.xamarin.MonoDevelop
+  </pre>
+
+  See also: http://www.monodevelop.com/download/linux/
+
+  ## <a name="hexchat"></a>Hexchat
+
+  <pre>
+  <span class="unselectable">$ </span>flatpak remote-add --if-not-exists gnome https://sdk.gnome.org/gnome.flatpakrepo
+  <span class="unselectable">$ </span>flatpak remote-add --if-not-exists tingping https://dl.tingping.se/flatpak/tingping.flatpakrepo
+  <span class="unselectable">$ </span>flatpak --user install tingping io.github.Hexchat
+  </pre>
+
+  ## <a name="gnome-twitch"></a>Gnome Twitch
+
+  <pre>
+  <span class="unselectable">$ </span>flatpak remote-add --if-not-exists gnome https://sdk.gnome.org/gnome.flatpakrepo
+  <span class="unselectable">$ </span>flatpak remote-add --if-not-exists tingping https://dl.tingping.se/flatpak/tingping.flatpakrepo
+  <span class="unselectable">$ </span>flatpak --user install tingping com.vinszent.GnomeTwitch
+  </pre>
+
+  ## <a name="pithos"></a>Pithos – Pandora Radio Client
+
+  <pre>
+  <span class="unselectable">$ </span>flatpak remote-add --if-not-exists gnome https://sdk.gnome.org/gnome.flatpakrepo
+  <span class="unselectable">$ </span>flatpak remote-add --if-not-exists tingping https://dl.tingping.se/flatpak/tingping.flatpakrepo
+  <span class="unselectable">$ </span>flatpak --user install tingping io.github.Pithos
+  </pre>
+
+  ## <a name="pithos"></a>Transmission Gtk
```

Comment:
Copy paste error with `'pithos` on this and the one below. Also the title for this one should be `Transmission Remote Gtk` to be clear it is a remote only client.